### PR TITLE
Add week 0 and week 1

### PR DESCRIPTION
**Pull request by menor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
-# waxon
-Front end training path
+# Wax On
+Training path to reach front end nirvana.
+
+## Week Zero
+- Ask your manager to buy this two books: [The Pragmatic Programmer](https://www.amazon.de/Pragmatic-Programmer-Journeyman-Master/dp/020161622X/ref=sr_1_1?s=books-intl-de&ie=UTF8&qid=1506408775&sr=1-1&keywords=pragmatic+programmer) and [Eloquent Javascript](https://www.amazon.de/Eloquent-JavaScript-Modern-Introduction-Programming/dp/1593275846/ref=sr_1_1?s=books-intl-de&ie=UTF8&qid=1506408862&sr=1-1&keywords=eloquent+javascript). Start reading the Pragmatic Programmer as soon as you get your hands on it.
```
Comment (sheley) [inline README.md]: > Ask your manager to buy this two books

"this" should be "these"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+- Identify your [learning style](http://vark-learn.com/the-vark-questionnaire/?p=questionnaire)
+- Identify your [thinking style](http://www.thelearningweb.net/personalthink.html)
+- Take a baseline [test](http://www.typingtest.com/index.html) to see how fast you currently type.
```
Comment (sheley) [inline README.md]: i'm a little skeptical that we care about how fast someone can type? i've never seen this included in anyone's "learn to code" resources and seems kind of unnecessary....

File: README.md [hunk 3]
```diff
@@ -1,2 +1,19 @@
...
+- Then practice. Try [typing.io](http://typing.io/) to practice typing code, which has a ton of strange symbols. You want to reach 40wpm.
```
Comment (sheley) [inline README.md]: this seems kinda more useful but won't someone learn how to type for programming by... programming?

again, i just think this kind of adds unnecessary worries. maybe we could include these two resources under a "bonus" section, to indicate that they are less important.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,19 @@
...
+- Pick a code editor, most popular are [Sublime](http://www.sublimetext.com/), [Atom](https://atom.io/) and [VSCode](https://code.visualstudio.com/)
+- Spend some time learning your editor and its plugin ecosystem, here are some useful plugins for [atom](https://github.com/mehcode/awesome-atom). **Don't** go crazy installing plugins (yet 😉)
```
Comment (sheley) [inline README.md]: we could add the link for how to install package control to get plugins for sublime text: https://packagecontrol.io/installation

i couldn't find an equivalent list that recommends my favorite sublime text packages but i could write a list of the packages i use.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,19 @@
...
+
+## Week One
+- Ask your manager for a pro account on [Codecademy](https://www.codecademy.com)  
+- Complete the [git](https://www.codecademy.com/learn/learn-git) and [command line](https://www.codecademy.com/learn/learn-the-command-line) courses
+- Lessons 1-6 from the [introduction to Javascript course](https://www.codecademy.com/learn/introduction-to-javascript)
+#### Reading:
+- Finish *The Pragmatic Programmer*
+- Chapters 1-5 of *Eloquent Javascript*
```
Comment (sheley) [inline README.md]: that is a lot to read in one week if i remember right, those chapters are pretty dense!

File: README.md (hunk 4 above)
Comment (sheley) [reply README.md]: ah now that you showed me, let's include this: https://github.com/dreikanter/sublime-bookmarks

File: README.md (hunk 5 above)
Comment (martinrp) [reply README.md]: Yeah, I agree, I think in general I also don't want to set specifics on how much to do week by week. For learning it should be largely self-defined, as people learn at different speeds. Also different workloads might come up, in regards to other duties, so don't want to make there too much pressure.

File: README.md (hunk 2 above)
Comment (menor) [reply README.md]: Yes makes sense, they made me learn but I had to do a lot of pair programming, so I will remove

File: README.md (hunk 5 above)
Comment (menor) [reply README.md]: Yep, makes sense

**Comment by sheley** [comment]
we can also add something before week 0 to encourage the use of something called a learning log in education terms- basically it's a short record that you keep of everything as you are learning. I did a writeup here since i can't open a PR since yours isn't merged yet. (Note that i adapted my text from https://www.ppd.admin.cam.ac.uk/files/learning_log.pdf ) 

### Keep a Learning Log

A learning log is a tool to record and reflect upon the time that you spend to achieve your learning objectives.

#### What is it for?

You'll reflect on what you actually learned and assess that against what you had hoped to learn. Through this process you can identify if there are any further gaps in your learning that you need to address. You'll also think about how what you learned can be applied in a practical way. 

When we undertake any kind of learning activity, unless we record it somewhere, we may forget about it. Keeping a detailed learning log can be a really useful resource for keeping track of your progress and as a reference in the future when you need a reminder about a particular topic. 

#### How it's done

Learning logs should be completed as soon as possible after each learning activity so that you can capture all the key learning points while they are still fresh in your mind. 

Your learning log can be kept in an actual notebook, or a spreadsheet/document, in the notes app, however and wherever you like. It only needs to take a few minutes at the start and end of the time you spend learning. It's just for you, so use any abbreviations you want (as long as you'll remember what they mean) and feel free to use incomplete sentences.

Before you begin your learning activity (reading the article/chapter, watching a tutorial video, or doing an exercise) record the following:

1. The date
2. What the activity is
3. What are you expecting to learn?

after you complete the activity answer the questions:

4. What have I learned?
5. How can I apply what I learned?

An example of how it might look is:

| Date | resource | What am I expecting to learn? | What have I learned? | How can I apply what I learned? |
| ------------- | ------------- |------------- |------------- |------------- |
| 24.9 | codecademy exercise about strings | what a string is, how they are used in javascript | strings are a "type", there are methods like `.concat` just for combining strings | now i can write code that creates and manipulates strings, like in the ada app when we create the string that says "Hello, {name}" |
| 25.9 | pg 24-27 in eloquent javascript | what the difference is between different variable declarations, what keywords mean | the difference between `const` and `let`, some words have special uses in javascript like `function`  | i know how to define variables that can have different values and variables that have constant values and can't be changed which i can try out in my practice TODO list app |

**Comment by menor** [comment]
That's a great idea @sheley can you please put it in a PR?

**Comment by sheley** [comment]
ja will do after this gets merged

**Pull request merged**